pkg/mapping/processors/assetinventory.go

```diff
@@ -85,7 +84,7 @@ func (p *AssetInventoryProcessor) Process(ctx context.Context, resourceMapping *
 
 	resourceScope, err := parseProject(resourceName)
 	if err != nil {
-		return err
+		return pmaperrors.Wrap(err)
```

Comment:
Why in here use `wrap` but in line 121, using `new`?

Reply:
Changed to use `new` in both cases.